src/hotspot/share/runtime/suspendResumeManager.cpp

```diff
@@ -81,15 +81,28 @@ void SuspendResumeManager::set_suspended(bool is_suspend, bool register_vthread_
   AtomicAccess::store(&_suspended, is_suspend);
 }
 
+void SuspendResumeManager::set_suspended_current_thread(int64_t vthread_id, bool register_vthread_SR) {
+#if INCLUDE_JVMTI
+  if (register_vthread_SR) {
+    assert(_target->is_vthread_mounted(), "sanity check");
+    assert(_target == JavaThread::current(), "should be current thread");
```

Comment:
So this assert should be outside the INCLUDE_JVMTI section - right?

Reply:
ough,
yes, should be outside INCLUDE_JVMTI 
However, I just can't understand how those methods could be called if JVMTI is not included. Do we still have any mechanisms to suspend thread without jvmti nowdays?